lms/djangoapps/discussion/tests/test_views.py

```diff
@@ -125,6 +125,7 @@ def make_mock_thread_data(
         group_id=None,
         group_name=None,
         commentable_id=None,
+        is_commentable_divided=None,
```

Comment:
Note to reviewers: it is necessary to pass is_commentable_divided in the response after creating a thread, else the visibility message is not correct. This bug exists on master today.